translations/messages+intl-icu.en.xlf

```diff
@@ -438,6 +438,59 @@
                 <source>paginator.current</source>
                 <target>(current)</target>
             </trans-unit>
+
+            <trans-unit id="reset_password.button">
+                <source>reset_password.button</source>
+                <target>Reset password</target>
+            </trans-unit>
+            <trans-unit id="reset_password.request">
+                <source>reset_password.request</source>
+                <target>Enter your email address and we we will send you a link to reset your password.</target>
+            </trans-unit>
```

Comment:
RE: naming conventions,

Would anyone object to having the translation `id`'s / `source` prefixed with `reset_password.` Personally I feel this makes it easier to maintain the translations for different parts of the app. Seems a bit easier to know where a translation is used. e.g. `reset_password.check_email.title` - translation is used in reset password check email template, and theres a pretty good chance it's the title translation for that page...